src/terminal/parser/stateMachine.cpp

```diff
@@ -1322,13 +1365,6 @@ void StateMachine::_EventOscString(const wchar_t wch)
 // - wch - Character that triggered the event
 // Return Value:
 // - <none>
-void StateMachine::_EventOscTermination(const wchar_t wch)
-{
-    _trace.TraceOnEvent(L"OscTermination");
-
-    _ActionOscDispatch(wch);

```

Comment:
This behaviour is effectively changed. Now we only dispatch the OSC string when `\\` or `BEL` is seen. Is there any negative impact?

Reply:
OK maybe I'm a little bit paranoid about changing the current code. It's a bugfix, right? It must change something to fix the bug. Would love someone to confirm this, though.